src/translation.cpp

```diff
@@ -22,9 +22,13 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "util/string.h"
 #include <unordered_map>
 
-// Per language translations
-std::unordered_map<std::string, Translations> all_translations;
-std::unordered_map<std::string, Translations> *g_all_translations = &all_translations;
+// Client translations
+Translations client_translations;
+Translations *g_client_translations = &client_translations;
```

Comment:
should be wrapped in `#ifndef SERVER`

Reply:
That also seems a bit complicated... Client translation used by EnrichedString, used in chat, which is used on server side, by TerminalChatConsole.
This would mean using a dummy Translations object for server compilation instead of existing client translation.

Reply:
...

Reply:
Finaly managed to do it :)